src/api/forms/service.js

```diff
@@ -40,6 +43,7 @@ export async function createForm(metadata, author) {
   // Validate the form definition
   const { error } = formDefinitionSchema.validate(definition)
   if (error) {
+    logger.warn(`Form failed validation: "${metadata.title}"`)
```

Comment:
Nice that we're logging the form title here 🙌

Do we need to make sure it's included in the next line too?

E.g. What if the Joi error says "'teamName' is invalid" but doesn't mention the form name?

```patch
    logger.warn(`Form failed validation: "${metadata.title}"`)
-   throw new InvalidFormDefinitionError(error.message, {
+   throw new InvalidFormDefinitionError(metadata.title, {
      cause: error
    })
```

Reply:
Done